### DISCUSSION TITLE
Feature ideas here!

### DISCUSSION THREAD

**Discussion by Inthenew** [Inthenew/Sketchbook#1] [General]
<h1>This is where feature ideas go!</h1>
<p>Put your ideas here and it might be added!</p>

**Comment by Dabombdgdjr** [comment]
add the ability to scale gravity in settings, would be cool

**Reply by Inthenew** [reply]
Will do

**Reply by Dabombdgdjr** [reply]
another thing, although I'm not sure how hard of a fix this would be, making your camera lock forward when you switch to first person perspective while driving. This is in contrast to the current car rotates around you situation.

**Comment by Eymens** [comment]
Hey man, I would like to convert sketchbook into game. Would you be interested in making it for a fair price?

**Reply by Eymens** [reply]
If you are interested you can contact me via email eymensahin@gmail.com

**Reply by Inthenew** [reply]
I sent an email! I am interested!

**Comment by washington254** [comment]
hey inthenew i would really love to implement the flight controls to React three Fiber and @react-three/cannon.. and would love your opinion. also this is really putting it out there but do you think it is possible to convert this whole project to React three fiber.. making each individual part into a seperate component so anyone can create this project by just importing </Sketchbook_Airplane> or </Sketchbook_Heli> into their project. thanks am going to build it on top of the existing sketchbook /"the one you have"/.. thanks

**Reply by Inthenew** [reply]
Hi, that sounds like a great idea! It should be possible to convert the entire project into React Three Fiber from what I know of, although it would be pretty challenging to make each part into a completely separate component but it should be possible.

**Reply by washington254** [reply]
hello, inthenew... yeah it's probably going to be difficult that's why am here to help... i specialize in React Three Fiber and i feel a lot of devs could benefit from these components ...i wanted to begin working on conversion but(you specified on the source code that there are files missing)..... let's talk here https://discord.gg/UqDfDbCj

**Comment by k4zol** [comment]
it would be nice if you added more of like a sketchbook-esk boat like a steam boat like this 
![boat1](https://github.com/user-attachments/assets/aea20d90-9edb-4fa9-8a50-83a0bfb9c14a)
or maybe add like a big boat with an interior like this one
![DSC_9620](https://github.com/user-attachments/assets/639599e1-93ac-47d0-8751-1117b1b558a4)

i feel like they give off a sketchbook feel its just i dont think that the jet skis fit into sketchbook very well. Tho i think they fit in with the boat race part maybe colour the jet skis white? Just to add the sketchbook feel.

also it would be nice if you expanded the boat race, and maybe like a easter egg if you go to far out into the ocean with a boat like maybe a tentacle will pull you under or you will start sinking or maybe if you go to far out with a plane it would cause severe turbulence eventually leading the player to crash

PS
thank you for continuing sketchbook, its truly a underrated masterpiece :)❤️

**Reply by Inthenew** [reply]
I like the idea of a boat with an interior! We'll see what I do but I agree the jet skis currently do not have that "sketchbook feel". I will probably make the jet skis mostly white and make a bigger boat with an interior, that does sound really cool.

**Reply by k4zol** [reply]
thanks for replying! i hope you take my ideas and put them in the game.
